src/sentry/event_manager.py

```diff
@@ -2789,9 +2787,7 @@ def _materialize_event_metrics(jobs: Sequence[Job]) -> None:
         job["event"].data["_metrics"] = event_metrics
 
         # Capture the actual size that goes into node store.
-        event_metrics["bytes.stored.event"] = len(
-            json.dumps_experimental("event-manager.enable-orjson", dict(job["event"].data.items()))
-        )
+        event_metrics["bytes.stored.event"] = len(orjson.dumps(dict(job["event"].data.items())))
```

Comment:
Nit: I'm not sure which one is more correct:

```suggestion
        event_metrics["bytes.stored.event"] = len(orjson.dumps(dict(job["event"].data.items())).decode())
```